charts/kong/values.yaml

```diff
@@ -823,6 +823,7 @@ resources: {}
   #  memory: 2G
 
 # readinessProbe for Kong pods
+# replace httpGet with exec if you want to use a custom command
```

Comment:
Can you please specify in the comments here an exemplary use case for exec? So that potential users know how exactly this can be used? I'm thinking something that could simply be uncommented to work ( assuming that `httpGet` section gets commented ). WDYT?